src/lib.rs

```diff
@@ -47,11 +49,11 @@
 //! backend requires the C library and development headers to be installed to build RFD. The package
 //! names on various distributions are:
 //!
-//! | Distribution    | Package name |
+//! | Distribution    | Installation Command |
 //! | --------------- | ------------ |
-//! | Fedora          | gtk3-devel   |
-//! | Arch            | gtk3         |
-//! | Debian & Ubuntu | libgtk-3-dev |
+//! | Fedora          | apt install gtk3-devel   |
+//! | Arch            | apt install gtk3         |
+//! | Debian & Ubuntu | apt install libgtk-3-dev |
```

Comment:
And here I've just edited the table column to Installation command instead of package name to be a bit clearer.